File: addons/hr_holidays/models/mail_message.py
# Part of Odoo. See LICENSE file for full copyright and licensing details.

from odoo import models


class MailMessage(models.Model):
    _inherit = 'mail.message'

    def _author_to_store_partner_fields(self):
        return super()._author_to_store_partner_fields() + ["im_status", "leave_date_to"]

File: addons/hr_holidays/static/tests/mock_server/mock_models/mail_message.js
import { mailModels } from "@mail/../tests/mail_test_helpers";

export class MailMessage extends mailModels.MailMessage {
    _author_to_store_partner_fields() {
        return [...super._author_to_store_partner_fields(), "im_status", "leave_date_to"];
    }
}
